### DISCUSSION TITLE
Allow for changing the background overlay when showing a Popup

### DISCUSSION THREAD

**Discussion by BeepBeepBopBop** [CommunityToolkit/Maui#2290] [New Feature Discussions]
It would be nice to be able to change the background overlay that appears behind a popup upon showing. 

Currently we can change the background color of the popup, but there is a partially transparent overlay that appears and we have no way of adjusting its opactity, color, or animation transition.

**Comment by ghost** [comment]
For reference, [new popup supports this](https://learn.microsoft.com/en-us/dotnet/communitytoolkit/maui/views/popup#setting-popup-defaults).
